### PR TITLE
updates to _terms.yaml file (part 1) for bioinfo nodes leeuwenhoek release

### PR DESCRIPTION
**Pull request by bullenc**
See [DAT-454](https://gdc-ctds.atlassian.net/browse/DICT-454) for summary of expected changes.

### CODE REVIEW

File: src/gdcdictionary/schemas/_terms.yaml [hunk 1]
```diff
@@ -6879,13 +6881,15 @@ wilms_tumor_histologic_subtype:
...
-      Generic name for the workflow used to analyze a data set.
+      The name of the workflow used to process and/or analyze a data set.
     termDef:
       term:
-      source:
-      cde_id:
-      cde_version:
-      term_url:
+      source: NCIt
+      cde_id: C181725
```
Comment (stilesj-uchicago) [inline src/gdcdictionary/schemas/_terms.yaml:6890]: Is there a resource we can leverage to confirm the cde_id?

File: src/gdcdictionary/schemas/_terms.yaml [hunk 2]
```diff
@@ -1959,9 +1959,10 @@ encoding:
       term: >-
         Encoding
       source: FastQC
-      cde_id:
+      cde_id: Encoding
```
Comment (wwysoc2) [inline src/gdcdictionary/schemas/_terms.yaml]: This looks a bit odd as a cde_id.  Is this something new or a misplaced term?  Same with "Total Sequences"

File: src/gdcdictionary/schemas/_terms.yaml [hunk 3]
```diff
@@ -7499,3 +7503,79 @@ extrathyroid_extension:
       cde_id: 3179452
       cde_version: 1.0
       term_url: "https://cadsr.cancer.gov/onedata/dmdirect/NIH/NCI/CO/CDEDD?filter=CDEDD.ITEM_ID=3179452%20and%20ver_nr=1"
+
+genome_doublings:
```
Comment (stilesj-uchicago) [inline src/gdcdictionary/schemas/_terms.yaml]: There should be a naming discussion here. My thoughts are that that plurals indicate collections (e.g. Array types) and singular represents a single value. 
Should this be named genome_doubling_event_count ?

Comment (wwysoc2) [reply src/gdcdictionary/schemas/_terms.yaml]: @ZhenyuZ what are your thoughts?

File: src/gdcdictionary/schemas/_terms.yaml (hunk 2 above)
Comment (bullenc) [reply src/gdcdictionary/schemas/_terms.yaml]: We draw the definitions for Encoding and Total Sequences from this [site](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/3%20Analysis%20Modules/1%20Basic%20Statistics.html), so there is no formal cde_id or NCIt code. This was mainly to address that these properties have links that are identified as 'null' in the Viewer page for read_group_qc node: https://docs.gdc.cancer.gov/Data_Dictionary/viewer/#?view=table-definition-view&id=read_group_qc  

Side note, these properties belong to a node that is not immediately found on the overall View page (https://docs.gdc.cancer.gov/Data_Dictionary/viewer/), therefore more for internal use.

File: src/gdcdictionary/schemas/_terms.yaml (hunk 1 above)
Comment (bullenc) [reply src/gdcdictionary/schemas/_terms.yaml:6890]: Yes, the NCIt browser: https://ncit.nci.nih.gov/ncitbrowser/ 

The site does go down from time to time FYI, I was having trouble with it today.

File: src/gdcdictionary/schemas/_terms.yaml (hunk 3 above)
Comment (bullenc) [reply src/gdcdictionary/schemas/_terms.yaml]: This is a good point, we do have have in our change request conventions that plural named properties should usually be reserved for array type properties: https://gdc-ctds.atlassian.net/wiki/spaces/GDC/pages/83525668/Change+Request+Conventions

File: src/gdcdictionary/schemas/_terms.yaml (hunk 2 above)
Comment (bullenc) [reply src/gdcdictionary/schemas/_terms.yaml]: @wwysoc2 came across that NCIt _did_ have entries for those two, updated their definitions and codes.

File: src/gdcdictionary/schemas/_terms.yaml (hunk 3 above)
Comment (wwysoc2) [reply src/gdcdictionary/schemas/_terms.yaml]: Per Zhenyu:  "genome_doubling" should be fine.

**Review by wwysoc2 — APPROVED** [review]
Change to "genome_doubling" and I think we are good

File: src/gdcdictionary/schemas/_terms.yaml (hunk 3 above)
Comment (bullenc) [reply src/gdcdictionary/schemas/_terms.yaml]: @stilesj-uchicago any additional review before I merge this thing?

**Review by stilesj-uchicago — APPROVED** [review]

File: src/gdcdictionary/schemas/_terms.yaml (hunk 3 above)
Comment (stilesj-uchicago) [reply src/gdcdictionary/schemas/_terms.yaml]: You're good to go.

**Pull request merged**